linkedin.py

```diff
@@ -3,15 +3,15 @@
 import constants
 import models
 import repository_wrapper
-import utils
 import warnings
+from utils import sleeper, url_generator, utils, logger
 from selenium import webdriver
 from selenium.common.exceptions import NoSuchElementException, TimeoutException
 from selenium.webdriver.chrome.service import Service as ChromeService
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
-from utils import prGreen, prRed, prYellow
+from logger.logger import prGreen, prRed, prYellow
```

Comment:
OPTIONAL: Remove direct usage of prGreen, prRed, prYellow outside of logger. 

This suggestion is optional and can be done in a separate PR